components/ethernet/include/esp_eth.h

```diff
@@ -45,6 +45,11 @@ typedef enum {
     ETH_MODE_FULLDUPLEX,
 } eth_duplex_mode_t;
 
+#define ETH_DMA_RX_BUF_NUM CONFIG_DMA_RX_BUF_NUM
```

Comment:
This header file should include sdkconfig.h so that the CONFIG_xxx values are guaranteed to be defined.